solarforecastarbiter/reference_forecasts/models.py

```diff
@@ -124,15 +179,32 @@ def solar_pos_calculator(): return solar_position
 
 def hrrr_subhourly_to_subhourly_instantaneous(latitude, longitude, elevation,
                                               init_time, start, end,
+                                              interval_label,
                                               load_forecast=load_forecast,
                                               *, __model='hrrr_subhourly'):
     """
     Subhourly (15 min) instantantaneous HRRR forecast.
     GHI, DNI, DHI directly from model.
     Max forecast horizon 18 or 36 hours (0Z, 6Z, 12Z, 18Z).
+
+    Parameters
+    ----------
+    latitude : float
+    longitude : float
+    elevation : float
+    init_time : pd.Timestamp
+        Full datetime of a model initialization
+    start : pd.Timestamp
+        Forecast start. Forecast is inclusive of this point.
+    end : pd.Timestamp
+        Forecast end. Forecast is exclusive of this point.
+    interval_label : str
+        Must be instant
```

Comment:
any issue with setting the default to 'instant' then?

Reply:
makes the API inconsistent